pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -35,17 +37,97 @@ import (
 	"kubevirt.io/kubevirt/pkg/util/net/ip"
 	migrationproxy "kubevirt.io/kubevirt/pkg/virt-handler/migration-proxy"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
 )
 
-func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineInstance) error {
+func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineInstance, options *cmdv1.VirtualMachineOptions) error {
+	interfacesToReconnect := interfacesToReconnect(options)
+	if len(interfacesToReconnect) != 0 {
+		if err := l.reconnectGuestNics(vmi, interfacesToReconnect); err != nil {
+			return err
+		}
+	}
+
 	if err := l.setGuestTime(vmi); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func interfacesToReconnect(options *cmdv1.VirtualMachineOptions) map[string]struct{} {
+	interfaceMigrationOptions := options.InterfaceMigration
+	ifacesToRefresh := map[string]struct{}{}
+	for ifaceName, migrationOption := range interfaceMigrationOptions {
+		if migrationOption != nil && migrationOption.Method == string(v1.LinkRefresh) {
```

Comment:
The link-refresh is generalized to all interfaces, even secondary ones that are not relevant to the migration.
In some cases, this will be an unwanted degradation.

I think there is a need to explain this in detail in the commit message.
The commit message at the moment has no information about what is done here.

Reply:
The commit message was updated. Done.